logical-comaprison-operators.js

```diff
@@ -6,35 +6,35 @@ TASK 1:
  // Have fun! // 😃
 ********************************************************************************/
 
-const exp1 = 10 >= 10; // TODO: ADD YOUR EVALUATION HERE -->
+const exp1 = 10 >= 10; // TODO: ADD YOUR EVALUATION HERE -- true>
 
-const exp2 = "dog" == "dog"; // TODO: ADD YOUR EVALUATION HERE -->
+const exp2 = "dog" == "dog"; // TODO: ADD YOUR EVALUATION HERE --true>
 
-const exp3 = true != false; // TODO: ADD YOUR EVALUATION HERE -->
+const exp3 = true != false; // TODO: ADD YOUR EVALUATION HERE --true>
 
-const exp4 = "10" === 10; // TODO: ADD YOUR EVALUATION HERE -->
+const exp4 = "10" === 10; // TODO: ADD YOUR EVALUATION HERE --fales>
 
-const exp5 = 5 > 4; // TODO: ADD YOUR EVALUATION HERE -->
+const exp5 = 5 > 4; // TODO: ADD YOUR EVALUATION HERE --true>
 
-const exp6 = null == undefined; // TODO: ADD YOUR EVALUATION HERE -->
+const exp6 = null == undefined; // TODO: ADD YOUR EVALUATION HERE --fales>
 
-const exp7 = "true" == true; // TODO: ADD YOUR EVALUATION HERE -->
+const exp7 = "true" == true; // TODO: ADD YOUR EVALUATION HERE -- true>
 
-const exp8 = "false" == false; // TODO: ADD YOUR EVALUATION HERE -->
+const exp8 = "false" == false; // TODO: ADD YOUR EVALUATION HERE -- true>
 
-const exp9 = NaN === NaN; // TODO: ADD YOUR EVALUATION HERE -->
+const exp9 = NaN === NaN; // TODO: ADD YOUR EVALUATION HERE -- true>
 
-const exp10 = !false || false; // TODO: ADD YOUR EVALUATION HERE -->
+const exp10 = !false || false; // TODO: ADD YOUR EVALUATION HERE --true>
 
-const exp11 = false && !false; // TODO: ADD YOUR EVALUATION HERE -->
+const exp11 = false && !false; // TODO: ADD YOUR EVALUATION HERE --fales>
 
-const exp12 = "apple" > "pineapple"; // TODO: ADD YOUR EVALUATION HERE -->
+const exp12 = "apple" > "pineapple"; // TODO: ADD YOUR EVALUATION HERE --true>
 
-const exp13 = "2" > "12"; // TODO: ADD YOUR EVALUATION HERE -->
+const exp13 = "2" > "12"; // TODO: ADD YOUR EVALUATION HERE --fales>
```

Comment:
This should evaluate to `true`.
Even though numerically 2 is less than 12, since we're comparing strings, the comparison is based on their lexicographical order. In this case, the string "2" is considered greater than the string "12" because "2" comes after "1".